liquibase-core/src/main/java/liquibase/resource/Resource.java

```diff
@@ -0,0 +1,49 @@
+package liquibase.resource;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URI;
+
+public interface Resource {
+
+    /**
+     * Returns the canonical searchPath relative path for this resource.
```

Comment:
I mean it's the path you would look this file up with in the search path. Like if the file is `com/example/file.txt` in the resource accessor `/tmp/project`, it would return `com/example/file.txt`. 

I updated the javadoc to `Returns the normalized, ResourceAccessor-relative path for this resource.` Does that make it make more sense?

Reply:
It does make more sense. Some examples would probably be most helpful.